ting_openformat.field.inc

```diff
@@ -522,6 +529,68 @@ function _ting_openformat_parse_analytic_content_element($element) {
 
 }
 
+/**
+ * Helper function to parse tracks
+ * @param $tracks
+ * @param null $entity
+ * @param array $options
+ * @return array|null
+ */
+function _ting_openformat_parse_tracks($tracks, $entity = NULL, $options = array()) {
+  if (!isset($tracks) || !isset($tracks[1]['track'])) {
+    return null;
+  }
+  $render_array = array();
+  foreach ($tracks as $element) {
+    $render_array = _ting_openformat_parse_tracks_element($element['track'][0]);
```

Comment:
Hvorfor løber du et array igennem hvor du bare overskriver $render_array konstant?

Reply:
Tja, det var den dovne løsning - der er altid max to arrays, og hvis der er to er det første ikke interessant i denne sammenhæng (indeholder en ledetekst, der kun bruges i netpunkt, men output fra OpenFormat er genbrugt). Jeg overvejer en anden løsning...